src/github/query.js

```diff
@@ -0,0 +1,61 @@
+/* eslint-disable camelcase */
+
+import { fetchJson, URL } from '../vendor/requests';
+import { Template } from '../vendor/Template';
+import { GMTDate as Date, ISO_8601 } from '../vendor/dates';
+import { selectFrom } from '../vendor/vectors';
+import { coalesce } from '../vendor/utils';
+
+// EXAMPLE https://github.com/mozilla-mobile/fenix/labels/eng%3Aperformance
+const GITHUB_ISSUES = new Template('https://api.github.com/repos/{{owner}}/{{repo}}/issues');
+
+const BURNDOWN = {
+  owner: 'mozilla-mobile',
+  repo: 'fenix',
+  labels: 'eng:performance',
+  filter: 'all',
+};
+
+
+async function getIssues({ timeDomain }) {
+  const { labels, filter } = BURNDOWN;
+  const since = Date.newInstance(timeDomain.min).format(ISO_8601);
+
+  const response = await fetchJson(URL({
+    path: GITHUB_ISSUES.expand(BURNDOWN),
```

Comment:
@airimovici   I dislike the fact that parameters (like `owner`, `repo`) are found in the URL.  I wonder if this template expansion should be included in the URL.